api/modules/emailSubscriptions.js

```diff
@@ -31,15 +31,53 @@ const emailSubscriptions = module.exports = (() => {
                             text: 'New users email: ' + record.fields.Email,
                         },
                     ]
-                
+
                 })
-            )                
+            )
         })
         .then(sendgridResponse => {
             res.json({ message: 'Success' })
         })
     }
-    //method to insert Date in airtable 
+
+    const unsuscribeUser = (req, res) => {
+
+        airtable.updateRecord({
+            tableName: 'Subscriptions',
+            id: req.id,
+            fieldData: {
+                'Unsuscribed': true,
+            }
+        })
+        .then(record => {
+            return (
+                sendgrid.sendMessage({
+                    recipient: record.fields.Email,
+                    msg: [
+                        {
+                            to: record.fields.Email,
+                            from: 'contact@setlife.education',
+                            subject: 'Setlife Newsletter',
+                            text: 'The unsubscription will take effect within 3 days',
+
+                        },
+                        {
+                            to: 'social@setlife.network',
+                            from: 'contact@setlife.education',
+                            subject: 'New Subscription',
+                            text: record.fields.Email + 'wants to unsuscribe',
```

Comment:
Change text to 
```
record.fields.Email + ' has sent a request to unsubscribe. The records on Airtable should be updated accordingly.'
```

Also FYI you can use template literals in cases like this if you prefer. Both of these examples are the same:
```
record.fields.Email + 'wants to unsubscribe'

`${record.fields.Email} wants to unsubscribe`
```